extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -212,3 +224,16 @@
   bottom: 0;
   right: 0;
 }
+
+@keyframes fadeIn {
```

Comment:
for both fadeIn and fadeOut, `opacity` alone is not enough to hide (it keeps the invisible item tappable). They need `visibility:hidden/visible` as well.